sources/org.osbuild.curl

```diff
@@ -81,6 +82,103 @@ SCHEMA = """
+    94: ("Authentication error", SourceErrorKind.CURL_CREDENTIALS),
+    # Security errors
+    35: ("A TLS/SSL connect error", SourceErrorKind.CURL_SECURITY),
+    51: ("The server's SSL/TLS certificate or SSH fingerprint failed verification", SourceErrorKind.CURL_SECURITY),
+    53: ("SSL crypto engine not found", SourceErrorKind.CURL_SECURITY),
+    54: ("Cannot set SSL crypto engine as default", SourceErrorKind.CURL_SECURITY),
+    58: ("Couldn't use the specified SSL cipher", SourceErrorKind.CURL_SECURITY),
+    59: ("Peer certificate cannot be authenticated with known CA certificates", SourceErrorKind.CURL_SECURITY),
+    63: ("Requested FTP SSL level failed", SourceErrorKind.CURL_SECURITY),
+    65: ("Failed to initialize SSL Engine", SourceErrorKind.CURL_SECURITY),
+    76: ("Problem with reading the SSL CA cert", SourceErrorKind.CURL_SECURITY),
+    79: ("Failed to shut down the SSL connection", SourceErrorKind.CURL_SECURITY),
+    82: ("Could not load CRL file, missing or wrong format", SourceErrorKind.CURL_SECURITY),
+    83: ("TLS certificate issuer check failed", SourceErrorKind.CURL_SECURITY),
+    84: ("The FTP PRET command failed", SourceErrorKind.CURL_SECURITY),
+    90: ("SSL public key does not matched pinned public key", SourceErrorKind.CURL_SECURITY),
+    91: ("Invalid SSL certificate status", SourceErrorKind.CURL_SECURITY),
+    # Internal errors
+    2: ("Failed to initialize", SourceErrorKind.CURL_INTERNAL_ERROR),
+    4: ("A feature or option that was needed to perform the desired request", SourceErrorKind.CURL_INTERNAL_ERROR),
+    27: ("Out of memory. A memory allocation request failed", SourceErrorKind.CURL_INTERNAL_ERROR),
+    38: ("LDAP cannot bind", SourceErrorKind.CURL_INTERNAL_ERROR),
+    39: ("LDAP search failed", SourceErrorKind.CURL_INTERNAL_ERROR),
+    61: ("Invalid LDAP URL", SourceErrorKind.CURL_INTERNAL_ERROR),
+    62: ("Maximum file size exceeded", SourceErrorKind.CURL_INTERNAL_ERROR),
+    64: ("Sending the data requires a rewind that failed", SourceErrorKind.CURL_INTERNAL_ERROR),
+    70: ("Illegal TFTP operation", SourceErrorKind.CURL_INTERNAL_ERROR),
+    71: ("Unknown TFTP transfer ID", SourceErrorKind.CURL_INTERNAL_ERROR),
+    72: ("File already exists (TFTP)", SourceErrorKind.CURL_INTERNAL_ERROR),
+    73: ("No such user (TFTP)", SourceErrorKind.CURL_INTERNAL_ERROR),
+    74: ("Character conversion failed", SourceErrorKind.CURL_INTERNAL_ERROR),
+    75: ("Character conversion functions required", SourceErrorKind.CURL_INTERNAL_ERROR),
+    78: ("An unspecified error occurred during the SSH session", SourceErrorKind.CURL_INTERNAL_ERROR),
+    87: ("Unable to parse FTP file list", SourceErrorKind.CURL_INTERNAL_ERROR),
+    88: ("FTP chunk callback reported error", SourceErrorKind.CURL_INTERNAL_ERROR),
+    93: ("An API function was called from inside a callback", SourceErrorKind.CURL_INTERNAL_ERROR),
+    95: ("HTTP/3 layer error", SourceErrorKind.CURL_INTERNAL_ERROR)
```

Comment:
This kind of mapping makes a lot of sense to me (without going into the details of exactly what it should be).